py/aou_workbench_client/concepts.py

```diff
@@ -0,0 +1,107 @@
+from aou_workbench_client.auth import get_authenticated_swagger_client
+from aou_workbench_client.config import all_of_us_config
+from aou_workbench_client.swagger_client.apis.concepts_api import ConceptsApi
+from aou_workbench_client.swagger_client.models.domain import Domain
+from aou_workbench_client.swagger_client.models.search_concepts_request import SearchConceptsRequest
+from aou_workbench_client.swagger_client.models.standard_concept_filter import StandardConceptFilter
+from ipywidgets import interactive
+import pandas as pd
+
+from IPython.display import display, HTML
+
+_DOMAIN_DICT = { 
+    '': None,
+    'Observation': Domain.OBSERVATION,
+    'Procedure': Domain.PROCEDURE,
+    'Drug': Domain.DRUG,
+    'Condition': Domain.CONDITION,
+    'Measurement': Domain.MEASUREMENT,
+    'Device': Domain.DEVICE,
+    'Race': Domain.RACE,
+    'Gender': Domain.GENDER,
+    'Ethnicity': Domain.ETHNICITY 
+}
+
+_STANDARD_CONCEPT_FILTER_DICT = { 
+    '': StandardConceptFilter.ALL_CONCEPTS,
+    'Standard concepts': StandardConceptFilter.STANDARD_CONCEPTS,
+    'Non-standard concepts': StandardConceptFilter.NON_STANDARD_CONCEPTS
+}
+
+# Common vocabularies. (There are others in the data but they don't get much
+# use.)
+_VOCAB_IDS = [ 
+    '', 
+    'ATC',    
+    'CPT4',
+    'DRG',
+    'HCPCS',
+    'ICD10CM',
+    'ICD10PCS',
+    'ICD9CM',
+    'ICD9Proc',
+    'ISBT',
+    'ISBT Attribute',
+    'LOINC',
+    'Multum',
+    'NDC',
+    'NDFRT',
+    'PPI',
+    'RxNorm',
+    'RxNorm Extension',
+    'SNOMED',
+    'SPL',
+    'VA Product'
+  ]
+
+_RESULT_FIELDS = [
```

Comment:
I thought about that, then figured there actually is no reason to make a dict at all since the only thing we do is iterate over this. (Either way you have to make this list of pairs.)